### DISCUSSION TITLE
Accessing metadata of local video file using getVideoMetadata

### DISCUSSION THREAD

**Discussion by CodyBontecou** [remotion-dev/remotion#1974] [Q&A]
Can `getVideoMetadata` take a local .mp4 file and output the video's duration? I can't get it to accept a local .mp4 relative path as the src string.

**Comment by CodyBontecou** [comment]
I got it working. Found this example: https://www.remotion.dev/docs/get-video-metadata#example, where it shows the explicit import occurs at the top rather than as a src param.

**Reply by JonnyBurger** [reply]
Thanks for self-closing the issue! We recommend putting the video into the `public/` folder now and importing it via `staticFile()`.
I have updated the docs now to reflect this! That being said, the other solution also works.

**Reply by CodyBontecou** [reply]
Oo gotcha. Thank you for the info.
